### DISCUSSION TITLE
Welcome to SIH2023 Discussions!

### DISCUSSION THREAD

**Discussion by DevSwayam** [TechieeGeeeks/SIH2023_CertifyBlocks#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xRoneet** [comment]
Roneet Yadav
GitHub I'd: Dev-X-Innovate

**Comment by Anuradha-Naidu** [comment]
Anuradha Naidu 
ID : Anuradha Naidu

**Comment by vwakesahu** [comment]
Vivek Sahu
Id: @vwakesahu
